File: contest/LeetCode_Weekly_Contest_202/Minimum_Operations_to_Make_Array_Equal.py
class Solution(object):
    def minOperations(self, n):
        """
        :type n: int
        :rtype: int
        """
        # mid = (1 + 2*(n-1)+1)/2 = n
        return sum(map(lambda idx: n - 2*idx-1, [i for i in xrange((n+1)>>1)]))
        

File: contest/LeetCode_Weekly_Contest_202/Minimum_Operations_to_Make_Array_Equal.java
class Solution {
    public int minOperations(int n) {
        int ret = 0;
        int mid = n; // (1 + 2 * (n-1) + 1) >> 1 
        for (int i = 0; i < (n+1)>>1; ++i) {
            int x = 2 * i + 1;
            ret += mid - x;
        }
        return ret;
    }
}
